lib/raidenclient/RaidenClient.ts

```diff
@@ -68,26 +71,34 @@ type ChannelEvent = {
   amount?: number;
 };
 
+type TokenPaymentResponse = {
+  secret: string;
+};
+
 /**
  * A class representing a client to interact with raiden.
  */
 class RaidenClient extends BaseClient {
   public readonly type = SwapClient.Raiden;
-  public readonly cltvDelta: number = 0;
+  public readonly cltvDelta: number = 1;
```

Comment:
The reason for this change is that the `SwapRequestPacket` will not validate otherwise:
https://github.com/ExchangeUnion/xud/blob/master/lib/p2p/packets/types/SwapRequestPacket.ts#L37